dashboard-frontend/src/pages/AccountsPage/InputToken.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { ChangeEventHandler } from 'react';
+import { classNames } from '../../helpers/strings';
+
+const InputToken = ({
```

Comment:
I noticed that we show these in plain text, which is fine for a first iteration but in general secret values like tokens and client secrets we should set the input type to password (potentially with the ability to unhide them)

Reply:
Sounds reasonable! created an issue for it https://github.com/vlab-research/vlab/issues/111